sdk/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -318,57 +330,59 @@ public static String getEndpoint() {
      * Set the client session endpoint
      */
     @SuppressWarnings("deprecation")
-    public static void setSessionEndpoint(final String endpoint) {
+    public static void setSessionEndpoint(@NonNull final String endpoint) {
         getClient().getConfig().setSessionEndpoint(endpoint);
     }
 
     /**
      * Set the client report endpoint
      */
     @SuppressWarnings("deprecation")
-    public static void setEndpoint(final String endpoint) {
+    public static void setEndpoint(@NonNull final String endpoint) {
         getClient().getConfig().setEndpoint(endpoint);
     }
 
     /**
      * Set the client report context
      */
-    public static void setContext(final String context) {
+    public static void setContext(@Nullable final String context) {
         getClient().setContext(context);
     }
 
     /**
      * Set the client report app version
      */
-    public static void setAppVersion(final String version) {
+    public static void setAppVersion(@NonNull final String version) {
         getClient().setAppVersion(version);
     }
 
     /**
      * Set the binary arch used in the application
      */
-    public static void setBinaryArch(final String binaryArch) {
+    public static void setBinaryArch(@NonNull final String binaryArch) {
         getClient().setBinaryArch(binaryArch);
     }
 
     /**
      * Return the client report app version
      */
+    @NonNull
     public static String getAppVersion() {
         return getClient().getConfig().getAppVersion();
     }
 
     /**
      * Return which release stages notify
      */
+    @NonNull
```

Comment:
It looks like me like this can be `null` as this annotation was not changed in the config branch:
https://github.com/bugsnag/bugsnag-android/pull/398/files#diff-18f38731c79b15200b4f49b6eb495a3cR305